src/scene/shader-lib/programs/lit-shader.js

```diff
@@ -50,13 +51,6 @@ const builtinVaryings = {
 };
 
 class LitShader {
-    /**
```

Comment:
why removing JSDocs from here? I think I added them so that we get member validation inside the function.

Reply:
This was not intended, probably just a bad merge.

Reply:
Fixed.